main.go

```
@@ -0,0 +1,32 @@
+	infra := Domain.NewPersonInfrastructure()
+	personUsecase := usecase.NewPersonUsecase(personRepo, infra)
+
+	personController := controller.NewPersonController(personUsecase)
+
+	personRouter := router.NewPersonRouter(personController)
+	router := gin.Default()
+
+	router.Use(cors.New(config.CorsConfig))
+
+	personRouter.SetUpRouter(router)
+	router.Run(":8080")
```

golangci-lint

Error return value of `router.Run` is not checked

(errcheck)